ucfcs/GrowthPlus
# Julio/embedded objects

This PR implements embedded objects for the child schema, a quiz schema and removes the report schema. 
Embedded objects look similar to the main realm objects, but their purpose will be to keep track of the state of children in each roadmap/lesson. Embedded objects don't have primary keys and only exists within the lifecycle of the parent schema and its data is unique to each parent schema. 
The main roadmap and lesson schemas will be generated from json files at the start of the app. 
The json importer is only importing roadmap data. 

To test:
Wipe down the data from the emulator to start with a clean realm table. 
Add a few children, then open up the realm file, inspect that the new data is populated and notice the embedded objects for each child. 
Inspect that the data for Roadmap and lessons are being populated. 

Note:
You might notice a bug right after creating a child, the landing page is not being populated with new data, Don't try to add the new child again.  This is because the UI is not refreshing after adding a new child to realm, currently looking into a solution for this, to see the data, go to the parent portal and come back to the landing page.
